hubble-relay/cmd/serve/serve.go

```diff
@@ -184,7 +189,8 @@ func runServe(vp *viper.Viper) error {
 	}
 	gopsEnabled := vp.GetBool(keyGops)
 	if gopsEnabled {
-		if err := agent.Listen(agent.Options{}); err != nil {
+		addr := fmt.Sprintf("localhost:%d", viper.GetInt(keyGopsPort))
```

Comment:
This won't work as there is no global viper instance and this the port will always be `0`. Please, use `vp` instead.
```suggestion
		addr := fmt.Sprintf("localhost:%d", vp.GetInt(keyGopsPort))
```

Reply:
Indeed, thanks a lot for the catch!